modules/sigprofiler/1.0/config/default.yaml

```diff
@@ -30,10 +70,10 @@ lcr-modules:
 
         pairing_config:
             genome:
-                run_paired_tumours: True
-                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours: False
```

Comment:
Why was this changed? Won't this make it harder to find the input files?

Reply:
I was trying to figure out a way to get around an error but I can just include general.yaml config file. The error had to do with an unmatched normal not being provided.